hack/e2e.sh

```diff
@@ -153,6 +154,7 @@ function e2e::image_load() {
     for n in ${names[@]}; do
         $KIND_BIN load docker-image --name $CLUSTER ${DOCKER_REGISTRY}/$n:$IMAGE_TAG
     done
+    $KIND_BIN load docker-image --name $CLUSTER ${DOCKER_REGISTRY}/pingcap/e2e-helper:latest
```

Comment:
we could add pingcap/e2e-helper into `names`

```sh
local names=(
        pingcap/chaos-mesh
        pingcap/chaos-daemon
)
```

Reply:
I put it below since I think maybe helper image just needs to use fixed `latest` tag. Do we need to use `${IMAGE_TAG}`? 
Do we need to push it to docker hub as well? If yes, I am happy to update the tag

Reply:
Someday we would use e2e test framework as our stability env and we would push relative image to the internal hub.